mappings/net/minecraft/util/JeonHelper.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_3518 net/minecraft/util/JsonHelper
+CLASS net/minecraft/class_3518 net/minecraft/util/JeonHelper
```

Comment:
`Jeon`? is that correct?

Reply:
Oh I think he named it `JsonUtil` before and decided to revert to `Helper` somehow 🤔